.gitignore

```diff
@@ -1,5 +1,7 @@
-test/errlog
-test/outlog
+/test/errlog
+/test/outlog
```

Comment:
Non absolute paths are matched anywhere in the repo. Presumably, we wouldn't want to arbitrarily ignore `/foo/test/outlog` for whatever reason.